pkgs/applications/science/logic/isabelle/default.nix

```diff
@@ -1,7 +1,8 @@
-{ lib, stdenv, fetchurl, perl, perlPackages, makeWrapper, nettools, java, polyml, z3, veriT, rlwrap, makeDesktopItem }:
+{ lib, stdenv, fetchurl, nettools, java, polyml, z3, veriT, vampire, eprover, rlwrap, makeDesktopItem }:
 # nettools needed for hostname
-
-stdenv.mkDerivation rec {
+let
+  eprover-ho = eprover.override { enableHO = true; };
```

Comment:
Can you please put this in `all-packages.nix` as well?  The override is useful in general.

Reply:
done